lib/manager.js

```diff
@@ -40,66 +40,9 @@ var expandRequestUrlInput = function(sites) {
 // expectedText property out into an arry of objects
 // if it is not already an array of objects.
 var expandExpectedTextInput = function(sites) {
-  var err;
-  return _.map(sites, function(site){
-    if(site.expectedText) {
-      // We expect site.expectdText to be one of the
-      // following:
-      // 1. a string: "some search text"
-      // 2. an object: {text: "some search text", caseSensitive (optional): true/false }
-      // 3. an array: Of 1. or 2. or a mixture of them.
-      //
-      // First step, if it's not an array then we need to get it into a 1 element array
-      // for the rest of the function.
-      if(!_.isArray(site.expectedText)){
-        site.expectedText = [site.expectedText];
-      }
-      // Now we know that it's an array of 1 or more "somethings." We'll iterate through
-      // that and make each of them objects and throw errors for anything we can't handle
-      // or recognize
-      site.expectedText = _.map(site.expectedText, function(expectedText){
-        if(_.isObject(expectedText)) {
-          // We have an expectation that if this is already an object that it has at a minimum
-          // the 'text' property
-          if(!expectedText.text) {
-            err = new Error('text property is missing from expectedText object.');
-            err.expectedText = expectedText;
-            throw err;
-          }
-        } else if(_.isString(expectedText)) {
-          expectedText = { text: expectedText };
-        } else {
-          err = new Error('Unknown/Unexpected type for text property.');
-          err.expectedText = expectedText;
-          err.type = typeof expectedText;
-          throw err;
-        }
-        return expectedText;
-      });
-    }
-    return site;
-  });
 };
 
 var run = function(runData, settings, outAdapter, verbose) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['verbose' is defined but never used.](https://www.codacy.com/app/kevin_1379/codacy/file/1533868191/issues/source?bid=3007997&fileBranchId=3010243#l45)